js/components/controlBar.js

```diff
@@ -403,7 +417,8 @@ var ControlBar = React.createClass({
         key="fullscreen"
         tabIndex="0"
         aria-label={fullscreenAriaLabel}>
-        <Icon {...this.props} icon={fullscreenIcon} style={dynamicStyles.iconCharacter} />
+        <Icon {...this.props} icon={fullscreenIcon} style={dynamicStyles.iconCharacter}
+          className="oo-tooltip oo-tooltip-left" dataTip={fullscreenTooltip}/>
```

Comment:
`oo-tooltip-left` should be dynamic based on whether the button is the last element in the control bar. The user can customize the elements in the control bar via the skin.json, so fullscreen will not always be the last element.